src/consensus/aft/test/driver.cpp

```diff
@@ -24,12 +24,28 @@ constexpr auto shash = ds::fnv_1a<size_t>;
 int main(int argc, char** argv)
 {
   const regex delim{","};
-  string line;
   size_t lineno = 1;
   auto driver = shared_ptr<RaftDriver>(nullptr);
 
-  while (getline(cin, line))
+  if (argc < 2)
+  {
+    throw std::runtime_error(
+      "Too few arguments - first must be path to scenario");
+  }
+
+  std::ifstream fstream;
+  fstream.open(argv[1]);
+
+  string line;
+  while (getline(fstream, line))
```

Comment:
Making this take a filename rather than reading from stdin, so its easier to run through a debugger.

Reply:
My long term view is we would discard this frontend in favour of a bit of pybind11, because that's ultimately where we want to drive things from. Never did find the time for it though.